Gemfile

```diff
@@ -2,5 +2,6 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
+gem "byebug"
```

Comment:
After reviewing `datura.gemspec` file for local testing, this line can be removed because the gem is already required via `datura.gemspec` which the last line of this file includes.

Reply:
If I remove this requirement, it breaks. the `require "byebug"` at the top of data_manager.rb doesn't work. If I take that line out, then I can't add `byebug` into any of the override files.